nvm.sh

```diff
@@ -1789,7 +1789,7 @@ nvm_get_make_jobs() {
   elif [ "_$NVM_OS" = "_sunos" ]; then
     NVM_CPU_THREADS="$(psrinfo | wc -l)"
   elif [ "_$NVM_OS" = "_aix" ]; then
-    NVM_CPU_THREADS="$(lsconf | command grep 'Number Of Processors:'| command awk '{print $4}')"
+    NVM_CPU_THREADS="$(pmcycles -m | wc -l)"
```

Comment:
Sorry, yes, the number of cores - in other words, what we want is to guarantee that there can always be at least one other process running at the same time as `make`.